modules/gapi/test/internal/gapi_int_pattern_matching_test.cpp

```diff
@@ -271,7 +271,7 @@ TEST(PatternMatching, TestMultiplePatternOuts)
     EXPECT_EQ(matching_test::V({dx_nh, dy_nh}), match.protoOuts());
 }
 
-TEST(PatternMatching, TestPreprocSplit3)
+TEST(PatternMatching, TestPrepResizeSplit3)
```

Comment:
let's drop the "Prep" part everywhere
```suggestion
TEST(PatternMatching, TestResizeToNCHW)
```

Reply:
Lets leave it, we may use --gtest_filter=*TestPrep*

Reply:
we can use `--gtest_filter=*TestPrep*` as well with the new naming, but OK